src/execution.rs

```diff
@@ -5,14 +5,17 @@
 // Please see the LICENSE-APACHE or LICENSE-MIT files in this distribution for license details.
 // ------------------------------------------------------------------------------------------------
 
-use std::collections::HashMap;
+mod variables;
 
 use anyhow::Context as ErrorContext;
 use thiserror::Error;
 use tree_sitter::QueryCursor;
 use tree_sitter::QueryMatch;
 use tree_sitter::Tree;
 
+pub use variables::Globals;
+use variables::*;
+
```

Comment:
nit: Can you mimic the `use` style down below?  Prefix these with `crate::`, add them to the section below (`rustfmt` will auto-sort them alphabetically for you), and list out the specific types that you're using instead of using a `*` import.